### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by tidalf**

### CODE REVIEW

File: resources/lib/qobuz/api/raw.py [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
 import hashlib
 import math
-import pprint
-import requests
 import socket
 import sys
+from itertools import izip, cycle
+from time import time
+
+import requests
```
Comment (showi) [inline resources/lib/qobuz/api/raw.py]: my sort 😭 why ws :)

File: resources/lib/qobuz/api/raw.py [hunk 2]
```diff
@@ -197,8 +197,7 @@ def track_getFileUrl(self, intent="stream", **ka):
             'request_ts': ka['request_ts'],
             'request_sig': str(
                 hashlib.md5('trackgetFileUrlformat_id' + str(ka['format_id']) +
-                            'intent' + intent + 'track_id' + str(ka[
-                                'track_id']) + str(ka['request_ts']) + self.s4)
+                            'intent' + intent + 'track_id' + str(ka['track_id']) + str(ka['request_ts']) + self.s4)
```
Comment (showi) [inline resources/lib/qobuz/api/raw.py]: ugly part with the + will be much cleaner with '{}'.format(...)

File: resources/lib/qobuz/dog.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
     _allowed_keys[key] = re.compile(value)
 
 
-class dog():
+class dog:
```
Comment (showi) [inline resources/lib/qobuz/dog.py]: old style class ? 
```python
class dog(objec):
  pass
```

Comment (tidalf) [reply resources/lib/qobuz/dog.py]: chepo moi :)

File: resources/lib/qobuz/node/favorite.py [hunk 4]
```diff
@@ -309,13 +309,11 @@ def add_tracks(self, track_ids):
...
 
     def _delete_cache(self):
-        keys = []
-        keys.append(
-            cache.make_key(
-                '/favorite/getUserFavorites',
-                user_id=user.get_id(),
-                limit=self.limit,
-                offset=self.offset))
+        keys = [cache.make_key(
```
Comment (showi) [inline resources/lib/qobuz/node/favorite.py]: nice one .p

File: resources/lib/qobuz/dog.py (hunk 3 above)
Comment (showi) [reply resources/lib/qobuz/dog.py]: oki in python 3
```
class dog: pass
```

**Pull request merged**